cpp/refactor_module/algorithm/refactor.cpp

```diff
@@ -1 +1,64 @@
 #include "refactor.hpp"
+
+void Refactor::TransferProperties(const mgp::Node &node, mgp::Relationship &rel) {
+  for (auto &[key, value] : node.Properties()) {
+    rel.SetProperty(key, value);
+  }
+}
+
+void Refactor::Collapse(mgp::Graph &graph, const mgp::Node &node, const std::string &type,
+                        const mgp::RecordFactory &record_factory) {
+  if (node.InDegree() != 1 && node.OutDegree() != 1) {
```

Comment:
`indegree !=1 or outdegree != 1?`

Reply:
both in degree and outdegree must be 1, if we are to be consistent with neo4j usage

Reply:
yep, good spotting